src/ontology/cl-edit.owl

```diff
@@ -3244,6 +3246,13 @@ AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/cl#general_cell
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/cl#general_cell_types_upper_slim> "general_cell_types_upper_slim")
 SubAnnotationPropertyOf(<http://purl.obolibrary.org/obo/cl#general_cell_types_upper_slim> oboInOwl:SubsetProperty)
 
+# Annotation Property: <http://purl.obolibrary.org/obo/cl#kidney_slim> (kidney_upper_slim)
```

Comment:
```suggestion
# Annotation Property: <http://purl.obolibrary.org/obo/cl#kidney_upper_slim> (kidney_upper_slim)
```

Reply:
If refactoring in Protégé, it would change in all other cases.